greenwave/tests/test_product_versions.py

```diff
@@ -5,6 +5,12 @@
 import pytest
 
 from greenwave import product_versions
+from greenwave.subjects.subject import Subject
+from greenwave.subjects.subject_type import GenericSubjectType
+
+
+def test_subject():
+    return Subject(GenericSubjectType('koji_build'), 'nethack-1.2.3-1.rawhide')
```

Comment:
Not sure about what is tested here, no assertion?

Reply:
Maybe the method naming is confusing, looks like it is testing subject.

Reply:
Oh, yes, I should rename this - it is only a helper function.